RecoVertex/BeamSpotProducer/src/BeamFitter.cc

```diff
@@ -24,6 +24,10 @@ ________________________________________________________________**/
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
+
+// HOOK RM
+#include "RecoVertex/BeamSpotProducer/interface/BeamSpotWrite2Txt.h"
```

Comment:
Ahem... it's me and it's not relevant, sorry.
I can PR into Giovanni's branch and if he merges the changes land directly here. Ok Slava?